docs/mms_on_fargate.md

```diff
@@ -0,0 +1,256 @@
+```
+
+So there is no need to pre-bake actual binary of the model to the container, you can just specify the HTTPS link to the binary.
+
+The last question that we need to address: how we should be starting our MMS within our container. And the answer is very simple, you just need to set the following ENTRYPOINT ([what is ENTRYPOINT?](https://docs.docker.com/engine/reference/builder/#entrypoint): 
+
+```bash
+mxnet-model-server start --mms-config /mxnet-model-server/mms_app_cpu.conf
+```
+
+And this it, nothing else.
+
+At this point I think we are ready to start creating actual Task definition.
+
+## Create a SqueezeNet Task Definition
+
+This is the first task where we finally ready to start doing something:
+
+1. Log-in to the AWS console and go to the Elastic Cloud Service / Task Definitions and press “Create new Task Definition”:
+
+![task def](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/1_Create_task_definition.png)
+
+2. Now you need to specify type of the task, surprise-surprise, we will be using the Fargate task:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/2_Select_Fargate.png)
+
+3. Task requires some configuration, let's look on it step by step, first set the name:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3_Config_1.png)
+
+Now is important part, one need to create [IAM role](https://aws.amazon.com/iam) that will be used to publish metrics to CloudWatch:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/Task+Execution+IAM+Role+.png)
+
+Our containers are optimized for 8 vCPUs, however in our example we going to use slightly smaller task with 4 vCPUs and 8 GB of RAM:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/cpu+and+ram.png)
+
+2. Now it is time to configure the actual container that the task should be executing
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/container+step+1.png)
+
+3. Next one need to specify port mapping, we need to expose port 8080. This is the port that our container is listening to. If needed it can be configured via the config [here](https://github.com/awslabs/mxnet-model-server/blob/master/docker/mms_app_cpu.conf#L40).
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/port+8080.png)
+
+Now the health checks. MMS has pre-configured endpoint /ping that can be used for health checks.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/container+health+checks.png)
+
+and the environment, with the entry point that we have discussed earlier:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/entrypoint.png)
+
+Everything else can be left as default. So feel free to press add and finish the creation of our very first task. If everything is ok you should now be able to see your task in the list of task definitions.
```

Comment:
press --> click

`Add` or say it's a button.

and --> to finish

our --> your

ok,